appengine/pond_storage/get.py

```diff
@@ -19,18 +19,100 @@
 
 import cgi
 import json
+import logging
 from google.appengine.ext import ndb
+from google.appengine.api import users
 from pond_storage import *
 
+
+def format_top_ducks(top_ducks_info):
+  """Adds an isOwner property to top ducks that are owned by the current user.
+  Args:
+    top_ducks_info: A list of ducks with the best rankings.
+  Returns:
+    A list of dictionaries containing duck information.
+    Example:
+    {
+      'name': 'bob',
+      'duck_key': 'urlsafe duck key',
+      'code': {'js': 'some code', 'opt_xml': 'some xml'},
```

Comment:
I don't think that top ducks should return code. It should only return what is needed for the request, especially since user code is more sensitive information. The duck_info methods could be refactored to allow for customizing how much info to include, or I would like at least a TODO for it.

Reply:
Added code to specify which keys you want.